schema.sql

```diff
@@ -1,11 +1,34 @@
-/* Database schema to keep the structure of entire database. */
+-- These scripts will delete the tables if it's already there
+DROP TABLE IF EXISTS owners;
+DROP TABLE IF EXISTS species;
+DROP TABLE IF EXISTS animals;
```

Comment:
- [ ] (Optional) I would advise you to drop tables and their dependents whenever you want to recreate the table. To do this, you need to use the key work _**CASCADE**_. `DROP TABLE IF EXISTS owners CASCADE`. 👍